src/mesh_tools.rs

```diff
@@ -146,8 +296,8 @@ pub fn weld(geometry: &Geometry, tolerance: f32) -> Geometry {
 /// Crawls the geometry to find continuous patches of geometry.
 /// Returns a vector of new separated geometries.
 pub fn separate_isolated_meshes(geometry: &Geometry) -> Vec<Geometry> {
-    let face_to_face = face_to_face_topology(geometry);
-    let mut available_face_indices: HashSet<u32> = face_to_face.keys().copied().collect();
+    let face_to_face = mesh_topology_analysis::face_to_face_topology(geometry);
+    let mut available_face_indices: HashSet<u32> = face_to_face.keys().cloned().collect();
```

Comment:
Why was this changed from `copied` to `cloned`?

Reply:
not sure. will set it back to `copied`